{"name": "Login Test with Different Credentials", "status": "passed", "attachments": [{"name": "Login Error Message", "source": "19f475e3-d7d2-449e-91dc-c4ef30e80d36-attachment.txt", "type": "text/plain"}, {"name": "Login Failure Screenshot", "source": "ef398c29-b932-41f9-9f88-118e43747989-attachment.png", "type": "image/png"}], "parameters": [{"name": "username", "value": "'locked_out_user'"}, {"name": "password", "value": "'secret_sauce'"}], "start": 1744806166126, "stop": 1744806179001, "uuid": "9c3d7120-2499-485d-a15d-4803ed043244", "historyId": "d29707f3907ee1fd9ae2cf349171da56", "testCaseId": "a707b9555172d26c5c1ec2d556e674f3", "fullName": "tests.test_login#test_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "Kaustubh-Nitro5"}, {"name": "thread", "value": "16980-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}